R/Rules-methods.R

```diff
@@ -664,8 +664,7 @@ setMethod("nextBest",
 ## ============================================================
 
 h_info_theory_dist <- function(probability,target,asymmetry){
-  delta = ((probability-target)^2)/((probability^asymmetry)*(1-probability)^(2-asymmetry)))
-  return delta
+  return ((probability-target)^2)/(((probability^asymmetry)*(1-probability)^(2-asymmetry)))
```

Comment:
```suggestion
  ((probability-target)^2)/(((probability^asymmetry)*(1-probability)^(2-asymmetry)))
```
(no explicit return needed in R , and style wise it is better not to do it)

Reply:
old habits, will fix.